### DISCUSSION TITLE
Will Github Codespaces ever be free for all?

### DISCUSSION THREAD

**Discussion by TechStudent10** [community/community#5050] [Codespaces] [answered]
According to a post, Github Codespaces will be free for eligible teams and organizations **until** September 10th. Does that mean that even though I can use Codespaces now by pressing "." on any repo, will that end after September 10th?

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @TechStudent11 
The experience around using '.' to open any GitHub repo in a VS Code web editor is different than Codespaces. This experience is available to all users and will continue to remain free. 

To share a bit more context on differences between Codespaces and web-based editor experience: 

- GitHub Codespaces is a cloud-powered development environment. With Codespaces you can get access to a full development environment running on a VM, with a full Visual Studio Code experience powered by GitHub’s cloud servers, and connect to that using a browser or the VS Code desktop app. 
- The web editor offers a browser-based experience to view and make light edits to code. While you cannot run or debug your code with the web version, you can use a subset of VS Code features and extensions to browse code repositories (quickly, without having to clone them) and make simple code changes.

Hope this helps clarify :)

**Reply by TechStudent10** [reply]
@tanmayeekamath Ohhhhhhhhhhhhhhhhhhhhhhhhhhh! Okay! Thanks for clearing that up!

**Comment by Photosell** [comment]
![P1510676](https://github.com/community/community/assets/147310512/57504b37-bc2a-4b98-9490-db8fae77b95d)
